.gitignore

```diff
@@ -41,20 +40,20 @@ Thumbs.db
 *.swp
 
 # Python #
-#################
+##########
 *.pyc
 *.egg-info/
 __pycache__/
 *.py[cod]
 .env
 
 # Django #
-#################
+##########
 *.egg-info
 .installed.cfg
 
-# Unit test / coverage reports
-#################
+# Unit test / coverage reports #
+################################
```

Comment:
Ah, I just saw your other reductions, didn't realize it was tied to line length.